roles/rook/tasks/deploy-loadbalancer.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Deploy Ceph dashboard loadbalancer
+  kubernetes.core.k8s:
+    state: present
+    force: true
+    definition:
+      kind: Service
+      metadata:
+        name: "{{ rook_cluster_name }}-mgr-dashboard"
+        namespace: "{{ rook_namespace }}"
+        labels:
+          app: rook-ceph-mgr
+          rook_cluster: "{{ rook_cluster_name }}"
+      spec:
+        ports:
+          - name: dashboard
+            port: "{{ rook_loadbalancer_dashboard_port }}"
+            protocol: TCP
+            targetPort: "{{ rook_loadbalancerdashboard_target_port }}"
+        selector:
+          app: rook-ceph-mgr
+          rook_cluster: "{{ rook_cluster_name }}"
+          mgr_role: active
+        sessionAffinity: None
+        type: LoadBalancer
+
+- name: Deploy Ceph RGW loadbalancer
+  kubernetes.core.k8s:
+    state: present
+    force: true
+    definition:
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: "{{ rook_cluster_name }}-rgw-rgw"
```

Comment:
Should we use `{{ rook_cluster_name }}-rgw-{{ rook_cephobjectstore_default_name }}` here?

Reply:
I think even the rgw in the middle is derived from the objectstore name. Afaik it implicitly makes a zone and a realm based on the store name.